tests/lib/user.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+TEST_USER="test"
```

Comment:
I was wondering if we have a central definition of test user anywhere (to replace this one), turns out we don't and simply use "test" anywhere :(

Reply:
Part of the snapd-spread-tribal-knowledge.